Makefile

```
@@ -34,3 +46,10 @@ test:
 
 vet:
 	go vet src/*
+
+build/data/%: data/%.in
+	mkdir -p build
```

At first I thought this should be `build/data` but reading https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html you can automatically get the directory name of the target:
```suggestion
	mkdir -p $(@D)
```